src/lib/download.js

```diff
@@ -22,6 +30,14 @@ export default (filename) => {
       fullUrl,
       (response) => {
         let cur = 0
+        const responseType = response.headers['content-type']
+        if (!responseType.includes('audio')) {
```

Comment:
content type may vary so it's just best to use node's https://nodejs.org/api/fs.html#fs_fs_existssync_path to check if the file is there. You can also add a `statSync` check that will determine the weight of the file, in order to allow only files over a specific limit (driven by constants / env vars) - give precedence to checking existence first tho

Reply:
update: we'll use request headers to identify the status of the file